src/test/resources/pages/instructorFeedbackAllSessionTypes.html

```diff
@@ -956,7 +956,7 @@ <h5 class="col-xs-2" data-original-title="Select this option to use a custom tim
             First Session #1
           </td>
           <td>
-            <span data-original-title="The feedback session has been created, is visible, and has ended.<br ></span>The responses for this session are visible." data-placement="top" data-toggle="tooltip" title="">
+            <span data-original-title="The feedback session has been created, is visible, and has ended.<br>The responses for this session are visible." data-placement="top" data-toggle="tooltip" title="">
```

Comment:
`</span>` in the previous tooltip was probably an error, this change should be fine.